source/agora/test/Base.d

```diff
@@ -262,6 +262,26 @@ public class LocalRestTaskManager : TaskManager
     {
         geod24.LocalRest.sleep(dur);
     }
+
+    /***************************************************************************
+
+        Run an asynchronous task after the set time in LocalRest's
+        can set it to repeat or run it only once.
+
+        Params:
+        timeout = Determines the minimum amount of time that elapses before
+            the timer fires.
+        callback = If non-null, this delegate will be called when the timer
```

Comment:
You are not checking for `null` in the LocalRest change in https://github.com/TrustHenry/localrest/compare/89f3a04e1d87a2c3c45e81659598e0d72bcb7c54...541d4db05ccc68dc423d93d93128502739a36325#diff-b5229c55d46fb0a852c49674585a3595.

So the code will segfault with a null callback here.

Reply:
Yes, Good catch!